### PR TITLE
tma-639: improve logging

### PR DESCRIPTION
**Pull request by hacxman**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/432/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/163/ : SUCCESS in 28s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/469/ : SUCCESS in 30s

**Comment by panjan** [comment]
ok to merge

File: lib/gooddata/helpers/data_helper.rb [hunk 1]
```diff
@@ -113,8 +113,10 @@ def realize_s3(params)
         filename = Digest::SHA256.new.hexdigest(@options.to_json)
         bucket = s3_client.bucket(bucket_name)
         obj = bucket.object(key)
+        GoodData.logger.debug("Contents of file from S3:")
```
Comment (yenkins) [inline lib/gooddata/helpers/data_helper.rb]: Calls 'gooddata.logger' 2 times (DuplicateMethodCall)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/434/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/166/ : SUCCESS in 28s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/472/ : SUCCESS in 28s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/158/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/169/ : SUCCESS in 27s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/475/ : SUCCESS in 29s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1046/ : SUCCESS in 45m 42s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/802/ : SUCCESS in 25m 32s

**Pull request merged**